capactio/capact
# Filter out Action Status update event

## Description

Changes proposed in this pull request:

- Filter out Action Status update event
  * Add `Result{Requeue: True}` when Request shoul be requeued with a backoff
  * Add `Resukt{RequeueAftert: 1*time.Microsecond}` when Request should be
  requeued without backoff


## Testing

* Run the cluster with populator disabled:

  ```bash
  DISABLE_MONITORING_INSTALLATION=true ENABLE_POPULATOR=false make dev-cluster
  ```

* Make a typo in the bitnami Postgresql manifest. For example, change `{{steps` to `{{stepssss`.
* Populate manifests
* Create and action: 

  ```bash
  capact action create   cap.interface.database.postgresql.install   --name psql
  ```
  imput params:
  ```yaml
  superuser:
    username: capact
    password: capact-pass123
  defaultDBName: dbdb
   ```
* get the action status
  ```bash
  capact act get psql -o yaml
  ```
* monitor how fast retries are changing. The last few should take noticeable amount of seconds.

## Related issue(s)

Fixes: #423
